Modules/Filtering/BiasCorrection/test/itkN4BiasFieldCorrectionImageFilterTest.cxx

```diff
@@ -270,26 +301,16 @@ N4(int argc, char * argv[])
   maskImage = maskshrinker->GetOutput();
   maskImage->DisconnectPipeline();
 
-  // set the input image and mask image
+  // Set the input image and mask image
   correcter->SetInput(inputImage);
   correcter->SetMaskImage(maskImage);
 
   using CommandType = CommandIterationUpdate<CorrecterType>;
   typename CommandType::Pointer observer = CommandType::New();
   correcter->AddObserver(itk::IterationEvent(), observer);
 
-  try
-  {
-    correcter->Update();
-  }
-  catch (const itk::ExceptionObject & excep)
-  {
-    std::cerr << "Exception caught !" << std::endl;
-    std::cerr << excep << std::endl;
-    return EXIT_FAILURE;
-  }
+  TRY_EXPECT_NO_EXCEPTION(correcter->Update());
```

Comment:
Also missing `ITK_` prefix. Many other macros are missing it too.

Reply:
Thanks @dzenanz 🦅 . Fixed in 155b729.